src/Controller/HoldRequestController.php

```diff
@@ -0,0 +1,261 @@
+<?php
+namespace NYPL\Services\Controller;
+
+use NYPL\Services\ServiceController;
+use NYPL\Services\Model\HoldRequest\HoldRequest;
+use NYPL\Services\Model\HoldRequestResponse;
+use NYPL\Starter\Filter;
+use Ramsey\Uuid\Uuid;
+use Slim\Http\Request;
+use Slim\Http\Response;
+
+/**
+ * Class HoldRequestController
+ *
+ * @package NYPL\Services\Controller
+ */
+class HoldRequestController extends ServiceController
+{
+    /**
+     * @SWG\Post(
+     *     path="/v0.2/hold-requests",
+     *     summary="Create new hold request",
+     *     tags={"hold-requests"},
+     *     operationId="createHoldRequest",
+     *     consumes={"application/json"},
+     *     produces={"application/json"},
+     *     @SWG\Parameter(
+     *         name="NewHoldRequest",
+     *         in="body",
+     *         description="",
+     *         required=true,
+     *         @SWG\Schema(ref="#/definitions/NewHoldRequest")
+     *     ),
+     *     @SWG\Response(
+     *         response=200,
+     *         description="Successful operation",
+     *         @SWG\Schema(ref="#/definitions/HoldRequestResponse")
+     *     ),
+     *     @SWG\Response(
+     *         response="401",
+     *         description="Unauthorized"
+     *     ),
+     *     @SWG\Response(
+     *         response="404",
+     *         description="Not found",
+     *         @SWG\Schema(ref="#/definitions/ErrorResponse")
+     *     ),
+     *     @SWG\Response(
+     *         response="500",
+     *         description="Generic server error",
+     *         @SWG\Schema(ref="#/definitions/ErrorResponse")
+     *     ),
+     *     security={
+     *         {
+     *             "api_auth": {"openid offline_access api"}
+     *         }
+     *     }
+     * )
+     */
+    public function createHoldRequest()
+    {
+        $data = $this->getRequest()->getParsedBody();
+
+        $data['jobId'] = Uuid::uuid4()->toString();
```

Comment:
Can we abstract this functionality to some kind of JobService (i.e. not call `UUID` directly)? I'm concerned we might need to keep some of the existing functionality related to "Jobs" because HTC is using it, so it might be best to keep this functionality very swappable.